### DISCUSSION TITLE
display bug on revision history pagination

### DISCUSSION THREAD

**Discussion by phil-hotchkiss** [mia-platform/community#427] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.2.1

### Description

Hey there! I am seeing a display bug in the revision history. The pagination is not clickable because it is behind a row. See attached image: 

![image](https://github.com/mia-platform/community/assets/136851567/1ebca4cb-b7aa-4726-9389-79524bd7ab1b)

Functionality of the button works. I was able to use a document selector and click it using the DevTools console. 

### Actual Outcome

Display of the pagination behaviors is in the wrong location of the DOM. 

### Expected Outcome

Pagination is rendered at the bottom of the result set, so it is clickable.

**Comment by fredmaggiowski** [comment]
Hi there! 

Wow we definitely tested pagination, I don't see why this is happening.

Will look into it tomorrow and see if it's already fixed in the upcoming patch release

**Reply by phil-hotchkiss** [reply]
For what it's worth, I attempted in both Chrome and Firefox browsers and received the same behavior. Thanks @fredmaggiowski!

**Reply by fredmaggiowski** [reply]
Thanks for the attempt, will get back at you asap

**Reply by fredmaggiowski** [reply]
We can't seem to replicate the bug with our console v12.2.1, does it consistently happen regardless of the page you are in?

**Reply by phil-hotchkiss** [reply]
The only page I've noticed it on is the revision history.

**Reply by fredmaggiowski** [reply]
I made some other tests but could still could not reproduce the issue, may I ask you the following things:

 - what is your screen resolution?
 - if you go the microservices page, reload the entire page, and open the revision history modal is this still happening?
 - does this happen even by refreshing the website cache (I believe it will still happen since you already tested it with different browsers, but I'd give it a try)
 - do you have some extensions installed that may mess with the DOM? (same note as above, unless you have the same extensions on both browser)

**Reply by danielemarostica** [reply]
Hi @phil-hotchkiss,

While testing for this i noticed it only happens with the old navigation, as a temporary workaround you could enable the new navigation. You can enable it from the menu clicking your user icon -> Feature Preview -> New Look and Feel

I'll keep you updated on when the fix will be ready to be used on the old navigation as well

**Reply by fredmaggiowski** [reply, marked as answer]
Nice catch @danielemarostica 

We'll try to provide a fix in the upcoming releases. However @phil-hotchkiss I'd recommend working with the new look and feel as it will become the only one available in a few weeks (end of March/beginning of April).
